### PR TITLE
Adds Team Management page

### PR DESCRIPTION
**Pull request by tosbourn**
[Explains how we can manage teams within Github](https://github.com/barnardos/BAU/issues/18).

Adds a page describing how best to manage people and projects in Github.

### CODE REVIEW

**Comment by tosbourn** [comment]
@jeddy3 I've addressed the points we talked about.

File: team-management.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Team Management
```
Comment (jeddy3) [inline team-management.md]: Shall we use lower case "m" for consistency with the other pages like "Pull requests"?

File: team-management.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+We use Github's in built team management in order to control access to our
```
Comment (jeddy3) [inline team-management.md]: Shall we simplify this to 

"We use GitHub's inbuilt team management to control access to our repositories. It provides the level of control we require."

(note the H in GithHub).

File: team-management.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+repositories. This lets us keep good levels of control over who can edit code.
+
+## Overview
```
Comment (jeddy3) [inline team-management.md]: Shall we remove this heading? It feels redundant and, unlike the other headings, isn't action orientated.

File: team-management.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+We have three teams:
+
+* `product-team` – people in Barnardo's who don't fall into the following teams
```
Comment (jeddy3) [inline team-management.md]: Shall we invert this list as a reader only becomes aware of the other groups after-the-fact, so to speak?

---

* `product-developers` – people who actively develop projects for Barnardo's
* `product-managers` – people who manage, test, or have some active involvement in code without being developers
* `product-team` – people in Barnardo's who don't fall into the previous teams

---

i.e. finish with the catch-all.

File: team-management.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+* `product-managers` – people who manage, test, or have some active involvement in code without being developers
+* `product-developers` – people who actively develop projects for Barnardo's
+
+If you are a member of either `product-managers` or `product-developers` you are automatically a member of `product-team`.
+
+Outside of the notion of teams is the notion of _owners_ and _members_, the following
```
Comment (jeddy3) [inline team-management.md]: Shall we break this into shorter sentences?

---

Outside of the notion of teams is the notion of _owners_ and _members_. To manage teams you'll need to be an _owner_. Speak to a [current owner](https://github.com/orgs/barnardos/people?utf8=%E2%9C%93&query=+role%3Aowner) if you believe you should be one.

----

File: team-management.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+assumes you are an owner. If you are not an owner and feel it would make sense
+that you should be made one, speak to a [current owner](https://github.com/orgs/barnardos/people?utf8=%E2%9C%93&query=+role%3Aowner)
+
+## Adding a new person
+
+To add a new person to an existing team:
+
+* select the appropriate team [from this list of teams](https://github.com/orgs/barnardos/teams)
```
Comment (jeddy3) [inline team-management.md]: Shall we number [these steps](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps) and remove the leading line?

---

## Adding a new person

1. Select the appropriate team [from this list of teams](https://github.com/orgs/barnardos/teams).
2. Click on "Members".
3. Select "Add New Member".

---

File: team-management.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+* click on "Members"
+* select "Add New Member"
+
+## Removing a person
```
Comment (jeddy3) [inline team-management.md]: As above? Whilst removing "and" ("you don’t put ‘or’ or ‘and’ after the bullets").

---

## Removing a person

1. Find the appropriate [team member](https://github.com/orgs/barnardos/people).
2. Click the cog.
3. Select "Remove from Organisation".

---

File: team-management.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+To remove someone from a team:
+
+* find the appropriate [team member](https://github.com/orgs/barnardos/people)
+* click the cog and select "Remove from Organisation"
+
+## Creating a new project
```
Comment (jeddy3) [inline team-management.md]: As above, and change "access to read" to "read access" to align with "write access"?

---

## Creating a new project

By default, everyone inside of Barnardo's will have read access to projects.

There are five steps to granting a team write access to a project.

1. Visit the settings page for the project.
2. Click on "Collaborators & Teams".
3. Click on "Add a team".
4. Select the appropriate team.
5. Select the correct access for the team.

---

**Comment by tosbourn** [comment]
Great feedback @jeddy3, thanks!

**Comment by tosbourn** [comment]
@jeddy3 would appreciate your eyes when you get a chance.

**Comment by sl33p** [comment]
Looking GTM with those changes Rich suggested

File: team-management.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
+# Team management
+
+We use GitHub's inbuilt team management to control access to our repositories.
+It provides the level of control we require.
```
Comment (jeddy3) [inline team-management.md]: Shall we soft wrap these paragraphs to be consistent with the other markdown files?

File: team-management.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+If you are a member of either `product-managers` or `product-developers` you are automatically a member of `product-team`.
+
+Outside of the notion of teams is the notion of _owners_ and _members_.
+To manage teams you'll need to be an owner.
+Speak to a [current owner](https://github.com/orgs/barnardos/people?utf8=%E2%9C%93&query=+role%3Aowner) if you believe you should be one.
+
+## Adding a new person
+
+1. Select the appropriate team [from this list of teams](https://github.com/orgs/barnardos/teams)
```
Comment (jeddy3) [inline team-management.md]: Shall we full stops to the end of these list items as per the guidelines?

File: team-management.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+We have three teams:
+
+* `product-developers` – people who actively develop projects for Barnardo's
+* `product-managers` – people who manage, test, or have some active involvement in code without being developers
+* `product-team` – people in Barnardo's who don't fall into the previous teams
+
+If you are a member of either `product-managers` or `product-developers` you are automatically a member of `product-team`.
+
+Outside of the notion of teams is the notion of _owners_ and _members_.
```
Comment (jeddy3) [inline team-management.md]: Soft wrap this paragraph?

File: team-management.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+To manage teams you'll need to be an owner.
+Speak to a [current owner](https://github.com/orgs/barnardos/people?utf8=%E2%9C%93&query=+role%3Aowner) if you believe you should be one.
```
Comment (jeddy3) [inline team-management.md]: Shall we shorten this to: "Contact a [current owner](https://github.com/orgs/barnardos/people?utf8=%E2%9C%93&query=+role%3Aowner) if you believe you should be one."

File: team-management.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+
+## Adding a new person
+
+1. Select the appropriate team [from this list of teams](https://github.com/orgs/barnardos/teams)
+2. Click on "Members"
+3. Select "Add New Member"
+
+## Removing a person
+
+1. Find the appropriate [team member](https://github.com/orgs/barnardos/people)
```
Comment (jeddy3) [inline team-management.md]: Shall we full stops to the end of these list items as per the guidelines?

File: team-management.md [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+2. Click the cog
+3. Select "Remove from Organisation"
+
+## Creating a new project
+
+By default, everyone inside of Barnardo's will have read access to a project.
+
```
Comment (jeddy3) [inline team-management.md]: Do we want to explain what these steps are for?

```

There are five steps to granting a team write access to a project.

1. xxx

```

Comment (tosbourn) [reply team-management.md]: I removed that line because I didn't think it added anything – I will update the heading to reflect that is happening.
